partners_api/yandex_api.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include "partners_api/taxi_base.hpp"
+
+#include <string>
+
+namespace ms
+{
+class LatLon;
+}
+
+namespace taxi
+{
+namespace yandex
+{
+class RawApi
```

Comment:
Please, add a comment, it's not easy to understand why do we need Api and RawApi.